FWCore/FWLite/src/BareRootProductGetterBase.cc

```diff
@@ -0,0 +1,249 @@
+// -*- C++ -*-
+//
+// Package:     FWLite
+// Class  :     BareRootProductGetterBase
+//
+// Implementation:
+//     <Notes on implementation>
+//
+// Original Author:  Chris Jones
+//         Created:  Tue May 23 11:03:31 EDT 2006
```

Comment:
Given this won't be consistent with the git history. How about you extend the comment say that you copied the original.

Reply:
How about now (in both `.cc` and `.h`)?